cics-bundle-maven-plugin/pom.xml

```diff
@@ -59,58 +65,53 @@
     <dependency>
       <groupId>org.apache.maven.plugin-tools</groupId>
       <artifactId>maven-plugin-annotations</artifactId>
-      <version>3.5.2</version>
+      <version>3.6.4</version>
       <scope>provided</scope><!-- annotations are needed only to build the plugin -->
     </dependency>
     
     <!-- test -->
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.13.1</version>
+      <version>4.13.2</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest</artifactId>
-      <version>2.1</version>
+      <version>2.2</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
-      <artifactId>mockito-all</artifactId>
-      <version>1.10.19</version>
+      <artifactId>mockito-core</artifactId>
+      <version>4.3.1</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>com.github.tomakehurst</groupId>
       <artifactId>wiremock-jre8</artifactId>
-      <version>2.26.3</version>
+      <version>2.32.0</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.xmlunit</groupId>
       <artifactId>xmlunit-matchers</artifactId>
-      <version>2.6.3</version>
+      <version>2.9.0</version>
       <scope>test</scope>
     </dependency>
-    
   </dependencies>
 
-  <prerequisites>
```

Comment:
Is there a reason you're removing this prereq? I'm not entirely sure what it does, but I'm guessing something like seeing the minimum version of the maven runtime that we're compatible with

Reply:
It was clashing with the version we enforced in parent pom which was 3.5.4 and the tool was giving an error.

Reply:
Prereqs [got removed with Maven 3](https://maven.apache.org/pom.html#Prerequisites) - recommendation is to use enforcer plugin instead.

Reply:
Great, that is what we have now, so will consider this resolved too